base-theme/layouts/shortcodes/underline.html

```diff
@@ -0,0 +1,2 @@
+{{ $text := .Inner  | default "" }}
+<span class="underline">{{- $text -}}</span>
```

Comment:
I might worth changing this to {{- $text | markdownify -}} to  support markdown in the  underline block. This will prevent issues such as this one: https://github.com/mitodl/ocw-studio/issues/1106